spec/features/idv/doc_auth/socure_document_capture_spec.rb

```diff
@@ -32,7 +32,7 @@
 
   after(:all) { @user.destroy }
 
-  context 'happy path' do
+  context 'happy path', allow_browser_log: true do
```

Comment:
in an existing standard and hybrid flow spec can we add a expect check using fake_analytics for the happy path, network error, and invalid request? 🙏🏿  the invalid request might be the only spec that doesn't exist

Reply:
can we still add a check to the network error spec?
can we create a spec for an invalid request ... for example, if we pass the wrong api secret key?

can we add fake analytics checks to the hybrid flow specs?

Reply:
thanks for adding the analytics check to the hybrid spec.  Can you add also add the checks in the hybrid spec for the same scenarios as you did for the standard flow (ie: network error, invalid response(401), ...) ?